File: CMS-SPA/src/app/core/_models/car-manage-record.ts
import { Pagination } from "./pagination";

export class CarManageRecord extends Pagination {
  companyId: number;
  plateNumber: string;
  driverName: string;
  licenseNumber: string;
  signInDate: Date;

  tempNumber: string;
  carTempNumber: string;
  signInReason: string;
  goodsName: string;
  goodsCount: string;
  departmentId: number;

  contactPerson: string;
  sealNumber: string;
  driverSign: string;
  signOutDate: Date;
  guardName: string;

  carId: number;
  isConfirm :string;
  memo :string;

  constructor() {
    super();
  }
}

File: CMS-API/Models/CMS/CarManageRecord.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace API.Models.CMS
{
    public class CarManageRecord
    {
        public string CompanyId { get; set; }
        public string PlateNumber { get; set; }
        public string DriverName { get; set; }
        [Key]
        public string LicenseNumber { get; set; }
        [Key]
        public DateTime SignInDate { get; set; }

        public string TempNumber { get; set; }
        public string SignInReason { get; set; }
        public string GoodsName { get; set; }
        public string GoodsCount { get; set; }
        public string DepartmentId { get; set; }

        public string ContactPerson { get; set; }
        public string SealNumber { get; set; }
        public string DriverSign { get; set; }
        public DateTime? SignOutDate { get; set; }
        public string GuardName { get; set; }

        public string CarId { get; set; }
    }
}